ocs_ci/ocs/ui/deployment_ui.py

```
@@ -419,6 +447,173 @@ def create_storage_cluster(self):
+        if scheme not in available_schemes:
+            raise ConfigurationError(
+                f"Selected EC scheme {scheme} not found in the table. "
+                f"Available: {available_schemes}"
+            )
+
+        ref_scheme, ref_k, ref_m, ref_cap = parsed[0]
+        total_raw = ref_cap * (ref_k + ref_m) / ref_k
+        # Raw storage will be always in TiB; we do not expect GiB values in the EC table
+        logger.info(
+            f"EC table: derived total raw capacity = {total_raw:.2f} TiB "
+            f"(from reference scheme {ref_scheme})"
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)